lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -87,6 +87,35 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: {
+              type: 'integer',
+              minimum: 60,
+              maximum: 86400,
+            },
+            maximumRetryAttempts: {
+              type: 'integer',
+              minimum: 0,
+              maximum: 185,
+            },
+          },
+        },
+        deadLetterConfig: {
+          type: 'object',
+          properties: {
+            targetArn: {
+              anyOf: [
+                { $ref: '#/definitions/awsArn' },
+                { $ref: '#/definitions/awsCfImport' },
```

Comment:
I believe now these extra definitions are reduntant as it's covered by `awsArn` definition, so we can avoid using `anyOf` as well